code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -623,8 +623,26 @@
 	weapon_weight = GUN_TWO_HAND_ONLY
 	can_scope = TRUE
 	init_firemodes = list(
-		/datum/firemode/semi_auto/slower
+		/datum/firemode/burst/two/slow
 	)
+
+/obj/item/gun/energy/laser/aer9/focused/rynn
+	name = "\improper Hot-wired AER10 laser rifle"
+	desc = "A pre-war laser rifle prototype that has seen better day, known as a failed attempt at competing with the wattz 2000. Part of it's grip seems broken along it's barrel, and it has cloth wrapped around it's stock and grip."
+	ammo_type = list(/obj/item/ammo_casing/energy/laser/lasgun/hitscan/focused)
+	icon = 'icons/fallout/objects/guns/energy.dmi'
+	lefthand_file = 'icons/fallout/onmob/weapons/guns_lefthand.dmi'
+	righthand_file = 'icons/fallout/onmob/weapons/guns_righthand.dmi'
+	mob_overlay_icon = 'icons/fallout/onmob/backslot_weapon.dmi'
+	icon_state = "hotwiredlaser"
+	item_state = "hotwiredlaser"
+	weapon_class = WEAPON_CLASS_RIFLE
```

Comment:
Similarly, all the stuff from weapon class to firemodes is basically just restated things on the parent object and can be removed. 😸